src/views/Home/components/LostAndFoundCard/index.tsx

```diff
@@ -0,0 +1,221 @@
+            ? report.locationLost
+            : `${report.locationLost.slice(0, 15)}...`}
+        </Typography>
+      </Grid>
+      <Grid item xs={width >= 900 ? 4 : 12}>
+        <Typography align="center">{report.category}</Typography>
+      </Grid>
+    </>
+  );
+
+  return (
+    <Card>
+      <CardHeader
+        title={
+          <Grid container direction="row">
+            <Grid item xs={7}>
+              Lost and Found
+            </Grid>
+            <Grid item xs={5}>
+              <Button variant="contained" color="secondary" component={Link} to="/LostAndFound">
+                Lost and Found Page
+              </Button>
+            </Grid>
+          </Grid>
+        }
+        className="gc360_header"
+      />
+      <CardContent>
+        <Grid container direction="row" justifyContent="space-between">
+          <Grid item xs={6}>
+            <Button
+              variant="contained"
+              color="secondary"
+              component={Link}
+              to="/LostAndFound/missingitemform"
+            >
+              Report Lost
+            </Button>
+          </Grid>
+          <Grid item xs={6}>
+            <Button variant="contained" color="secondary" component={Link} to="/LostAndFound">
+              Report Found
+            </Button>
+          </Grid>
+        </Grid>
+        {activeReports?.length < 1
+          ? noReports
+          : width >= 900 && (
+              <Grid container spacing={2} className={styles.headerRow}>
+                <Grid item xs={4}>
+                  <Typography align="center">Date Lost</Typography>
+                </Grid>
+                <Grid item xs={4}>
+                  <Typography align="center">Location Lost</Typography>
+                </Grid>
+                <Grid item xs={4}>
+                  <Typography align="center">Description</Typography>
+                </Grid>
+              </Grid>
+            )}
+        {activeReports?.length > 3 ? (
+          <>
+            {activeReports.slice(0, 3).map((report) => (
+              <>
+                <Link to="/lostandfound">
+                  <Card
+                    className={`${styles.dataRow} ${width < 900 ? styles.mobileDataRow : ''}`}
+                    key={report.recordID}
+                  >
+                    <CardContent>
+                      <Grid container spacing={2} alignItems="center">
+                        {width < 900 ? (
+                          <>
+                            {/* Display two items per row on mobile */}
+                            <Grid item xs={3}>
+                              <Typography align="center">{formatDate(report.dateLost)}</Typography>
+                            </Grid>
+                            <Grid item xs={3}>
+                              <Typography align="center">
+                                {report.locationLost.length > 15
+                                  ? report.locationLost.slice(0, 15) + '...'
```

Comment:
There's a way to do the ... using pure CSS that is more reactive.  You can find examples of that on the data entry list page (and maybe the general user list page?).